docs/organizer-guide/meetups/starting.rst

```diff
@@ -14,176 +13,186 @@ It contains a lot of the knowledge that we have put here:
-We highly recommend watching this before starting a meetup!
+For more information on the Write the Docs worldview, see Eric Holscher's Introduction to the conference and community from Write the Docs NA 2016:
 
-.. note:: You can skip ahead to 1:30, since the beginning is mostly conference logistics.
+.. note:: The "meat" of the worldview starts at 1:30.
 
 .. raw:: html
 
     <iframe width="640" height="360" src="https://www.youtube.com/embed/Fmfs-iYqx5Q?list=PLmV2D6sIiX3U03qc-FPXgLFGFkccCEtfv" frameborder="0" allowfullscreen></iframe>
 
-Done? Let's go into details.
+Now for some details
 
 Launch your Group
 -----------------
 
-Write the Docs meetups are organized using `Meetup <http://www.meetup.com/>`_. Through Meetup, you can create events, contact Meetup members, promote your sponsors, and more. If you need help with Meetup fees, `contact us <mailto:conf@writethedocs.org>`_. We may be able to help.
+You can start a Write the Docs Meetup via `Meetup <http://www.meetup.com/>`_. Through Meetup.com, you can create events, contact Meetup members, promote your sponsors, and more. If you need help with Meetup fees, `contact us <mailto:conf@writethedocs.org>`_. We may be able to help.
 
 Create a Meetup Page:
 ~~~~~~~~~~~~~~~~~~~~~
 
-Go to `Meetup website <https://secure.meetup.com/create/>`_ and follow the instructions.
+Go to the `Meetup website <https://secure.meetup.com/create/>`_ and follow the instructions.
 
-a few tips:
+Tips:
 
-* **Topics**: they're keywords associated with your group that will help people find it. If you're not sure what to choose, you can try: Technical Writers, Documentation, Technical Writing, Technical communicators, Technical Documentation Editing, Writing, Document Management Systems, Software Development, Web Development, User Experience, UX Design, Open Source.
-* **Name**: our meetups are usually called "Write the Docs" + city name.
-* **Members' name**: meetup's organizers usually choose "Documentarians".
-* **Code of Conduct**: don't forget to add Write the Docs' :doc:`/code-of-conduct` to the ``page`` section of your meetup.
+* **Topics**: Choose your topics with an eye to the Write The Docs worldview. We want to include everyoen interested in software documentation, so include topics that help publicize your meetups to the developers and engineers in your area, such as Python, JavaScript, APIs, Perl, Ruby, UX, User Experience, as well as standard topics associated with techincal writing.
+* **Name**: Our meetup include the following standard name: "Write the Docs" <city name>, such as "Write The Docs PDX".
+* **Members' name**: Our standard name is "Documentarian".
+* **Code of Conduct**: Do include a code of conduct. For options, see the stadnard Write the Docs' :doc:`/code-of-conduct` or the abbreviated version used by `Write the Docs PDX <https://www.meetup.com/Write-The-Docs-PDX/pages/19618086/Code_of_Conduct/>`_
```

Comment:
`standard` is misspelled. +1 on this. We should standardize on a meetup/community CoC.